app/assets/javascripts/application.js

```diff
@@ -33,12 +33,14 @@ Kochiku.delayedRefresh = function(updateInfo) {
   setTimeout(function() {
     if($('input#refresh').is(':checked')) {
       $.getJSON(document.URL + '/modified_time', function( data ) {
-        var buildTime = new Date(Date.parse(data));
-        if(buildTime > updateInfo.renderTime) {
-          window.location.reload();
+        var buildTime = Date.parse(data);
+        var renderTime = updateInfo.renderTime
+        if(buildTime > renderTime) {
+          Kochiku.buildInfo.renderTime = buildTime
+          Kochiku.updateBuildParts(renderTime)
```

Comment:
I believe semicolons are optional in JS now but leaving them off does not match the style of the rest of the file.